api-tests/src/clients/cri-stub-client.ts

```diff
@@ -5,6 +5,25 @@ import {
 } from "../types/cri-stub.js";
 import config from "../config/config.js";
 
+const CREDENTIAL_ISSUER_SUBDOMAINS: Record<string, string> = {
```

Comment:
Strictly this should be `STUB_CREDENTIAL_ISSUER_SUBDOMAINS` I think - to avoid confusion with 'real' stubs.